EU4toV2/Source/EU4World/EU4Country.cpp

```diff
@@ -21,6 +21,7 @@ THE SOFTWARE. */
 
 #include "EU4Country.h"
 #include "Country/EU4GovernmentSection.h"
+#include "Country/EU4NationalSymbol.h"
```

Comment:
This is in the .h file, so shouldn't be needed here.

Reply:
indeed.